Library/OcCryptoLib/Ia32/Sha512AvxDummy.nasm

```diff
@@ -0,0 +1,29 @@
+; @file
+; Copyright (C) 2021, ISP RAS. All rights reserved.
+;
+; This program and the accompanying materials
+; are licensed and made available under the terms and conditions of the BSD License
+; which accompanies this distribution.  The full text of the license may be found at
+; http://opensource.org/licenses/bsd-license.php
+;
+; THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+; WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+;
+BITS 32
+
+section .bss
+global ASM_PFX(mIsAvxEnabled)
+ASM_PFX(mIsAvxEnabled): db 0
```

Comment:
I'm not sure a .bss section will work well with CLANGPDB and MSVC. EDK II seems to merge .bss and .data sections into a .data segment for ELFs, which is good, but I think those that compile to PE directly would create a separate .bss PE section, which wastes space. Can the declaration maybe just be moved to C so the compiler can choose where to place it?

cc @vit9696

Reply:
Second the idea of declaring the var in C.